src/components/src/ProductDisplayItem/productdisplayitem.main.tsx

```diff
@@ -587,9 +601,17 @@ class ProductDisplayItemMain extends Component<ProductDisplayItemMainProps, Prod
     return (
       <div className="product-image-carousel">
         <Slider {...settings}>
```

Comment:
I'll send you a message on slack with the mockup of this. Let's see if we can show thumbnails and arrows outside of the main image.